IdentityCore/src/broker_operation/request/browser_native_message_request/MSIDBrowserNativeMessageGetTokenRequest.m

```diff
@@ -58,6 +58,26 @@ + (NSString *)operation
     return @"GetToken";
 }
 
+#pragma mark - MSIDBrokerOperationRequest
+
+- (NSString *)localizedApplicationInfo
+{
+    // If they are nil or empty -- return display name.
+    if ([NSString msidIsStringNilOrBlank:self.clientId] && [NSString msidIsStringNilOrBlank:self.redirectUri])
+    {
+        return self.localizedCallerDisplayName;
+    }
+    
+    // Otherwise show clientId and redirect uri.
+    __auto_type clientId = self.clientId ?: NSLocalizedString(@"N/A", nil);
+    __auto_type redirectUri = self.clientId ?: NSLocalizedString(@"N/A", nil);
```

Comment:
@ameyapat's reviewer was able to catch this error 👏🏻
it should be `redirectUri` instead of `clientId`

```suggestion
    __auto_type redirectUri = self.redirectUri ?: NSLocalizedString(@"N/A", nil);
```